bears/general/LicenseCheckBear.py

```diff
@@ -0,0 +1,40 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='licensecheck',
+        use_stdout=True,
+        output_format='regex',
+        output_regex=r'.*: .*UNKNOWN',
+        result_message='No license found.')
+class LicenseCheckBear:
+    """
+    Attempts to check the given file for a license, by searching the start
+    of the file for text belonging to various licenses.
+    """
+    LANGUAGES = {'All'}
+    REQUIREMENTS = {DistributionRequirement(apt_get='devscripts',
+                                            dnf='licensecheck')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'License'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         licensecheck_lines: int=None,
+                         licensecheck_tail: int=None):
+        """
+        :param licensecheck_lines:
+            Specify how many lines of the file header should be parsed
+            for license information.(Default: 60)
```

Comment:
`--lines` was working for me when I set `--tail` to 0. It will show unexpected behaviour when I set it to defaults. :worried: I think we should let the users figure out this. What say @Makman2 ?